.github/workflows/main.yaml

```diff
@@ -30,8 +30,15 @@ jobs:
         uses: snok/install-poetry@v1.3
       - name: Install dependencies & project
         run: poetry install --no-interaction
+      # Download and extract build sources
+      - name: Download inputs / sources
```

Comment:
The change.

Reply:
- [ ] @ShahimEssaid I need help with [this](https://docs.google.com/document/d/1CR6FEmt2kxFb-te-X8Hn8fOIgQ61x9DZHSchCZnifrQ/edit?disco=AAABYNZ14C4) before I merge this PR.